examples/nginx-ldap-auth/Furyfile.yml

```diff
@@ -0,0 +1,3 @@
+bases:
+  - name: ingress/nginx-ldap-auth
+    version: v1.4.0-RC3
```

Comment:
Yes we have. I was thinking in something like `bumpversion` to do this kind of stuff because i think there are more places to replace the version. https://pypi.org/project/bumpversion/ 

What do you think about it?

Reply:
lgtm! cool project

Reply:
Added: https://github.com/sighupio/fury-kubernetes-ingress/pull/16/commits/2967f7596a4748056606d0d2af4511e475a88d13

You can bumpversion using: 

- `bumpversion --new-version=1.4.0-RC5 rc`
or 
- `bumpversion --new-version=1.4.0 minor`

Thanks!